src/System.IO.FileSystem/src/System/IO/Directory.cs

```diff
@@ -579,6 +579,11 @@ public static void Delete(String path, bool recursive)
             String fullPath = Path.GetFullPath(path);
             FileSystem.Current.RemoveDirectory(fullPath, recursive);
         }
+
+        public static string[] GetLogicalDrives()
```

Comment:
> We should not then harm the current implementation by having yet another way of providing a PAL implementation that conflicts.

How does this harm/conflict? Our shared code already provides a PAL for code. Why put a PAL on a PAL or not keep the PAL in common only when it's simple on some platforms?. I'd prefer to see the various implementations living in proximity rather than have to hunt down the variants when starting from Common. Additionally when I utilize elsewhere (such as Environment.GetLogicalDrives()) I would much rather call DriveInfoInternal.GetLogicalDrives() and be done with it- as opposed to duplicating the same code (currently simple, I grant you).